### PR TITLE
GraphNotification: safe property access

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
Fixes crash in notifications screen. Don't have repro, @matildepark can you confirm fix?

### CODE REVIEW

File: pkg/interface/src/views/apps/notifications/graph.tsx [hunk 1]
```diff
@@ -296,7 +295,7 @@ export function GraphNotification(props: {
     dm,
     singleAuthor
   );
-  const groupAssociation = useAssocForGroup(association.group);
+  const groupAssociation = useAssocForGroup(association?.group);
```
Comment (matildepark) [inline pkg/interface/src/views/apps/notifications/graph.tsx:298]: I don’t have repro either. But do these functions handle undefined well?

Comment (liam-fitzgerald) [reply pkg/interface/src/views/apps/notifications/graph.tsx:298]: yes

Comment (matildepark) [reply pkg/interface/src/views/apps/notifications/graph.tsx:298]: Well as per https://github.com/urbit/landscape/issues/799 it does appear to be this line in question. I just wonder when we are passing undefined as an association.

Comment (liam-fitzgerald) [reply pkg/interface/src/views/apps/notifications/graph.tsx:298]: I'm not sure, perhaps notifications aren't being cleaned up correctly on graph remove?

**Review by matildepark — APPROVED** [review]
Well since neither of us can repro and these are harmless nice-to-dos anyway, it seems fine to try this?

**Pull request merged**